dashboard_matrix/generate_test_matrix_data.py

```
@@ -0,0 +1,151 @@
+import urllib.parse
+import requests
+import re
+import urllib
+from store_data import *
```

Ruff (0.8.2)

`from store_data import *` used; unable to detect undefined names

(F403)